### PR TITLE
Adds instructions to use docker during developement

### PR DESCRIPTION
**Pull request by svalo**
This PR adds instructions to use Docker during developement of the website

### CODE REVIEW

**Comment by ilario** [comment]
This looks great :D 
A few observations:
Can you add indentation to the code after the `\`?
Do you think jekyll/minimal would be enough?
I believe 4000:4000 is less confusing than 8080:4000
Anyway I can't properly 'cause on my system I get errors like:
```
Could not find gem 'asciidoctor (~> 1.5.0)' in any of the gem sources listed in your Gemfile.
```

**Comment by svalo** [comment]
@ilario 
> Can you add indentation to the code after the \?

Sure

>Do you think jekyll/minimal would be enough?

Nope, I tried building the website with jekyll minimal but it errors out. In the minimal image many packages are removed before installing the required gems for the given project. I guess it's the reason you get that error  about `asciidoctor`. Can you try with `jekyll/jekyll`?

> I believe 4000:4000 is less confusing than 8080:4000

You definitivelly got a point, don't know why I choose 8080 in the firse place

Commit 0b060200e5321be98d81b04ec9ea19d387a24d78 addresses theese suggestions

**Comment by ilario** [comment]
> Can you try with jekyll/jekyll?

Yep, already tried with the same error, don't worry it's some messy situation on my system between packages installed via system packaging and Gems.

**Comment by svalo** [comment]
@ilario mmmh your system shouldn't matter here, docker will create a disposable container sharing only the folder with website with the phisical host. Does the container have internet access when running? If not that might be the problem. Is it able to download other missing gems? Is there a folder in the website folder containing gems? Just randoms things to check that are coming to my mind

**Comment by aparcar** [comment]
I tried it an get the following error: 
```
You have requested:
  jekyll ~> 3.5.0

The bundle currently has jekyll locked at 3.1.6.
Try running `bundle update jekyll`

If you are updating multiple gems in your Gemfile at once,
try passing them all to `bundle update`
```

**Comment by pedro-nonfree** [comment]
It works for me

```
 $ docker -v
Docker version 17.09.0-ce, build afdb6d4
```
2017-11-13-131413_1365x745_scrot

**Comment by aparcar** [comment]
`Docker version 1.13.1, build 092cba3` - I'm even running debian testing here ☠️

**Comment by brunovianna** [comment]
Worked for me. Although it wouldn't hurt to add that you have to run from lime-web/ for it to work...

**Comment by svalo** [comment]
@aparcar in my experience I always had to use docker's own repo on debian/ubunutu to get a decent user experience, the cost being install software which didn't go through the distro packager.
I'm not a ruby expert (which is why I choosed to use docker instead of having gems installed on my system) however it seems that the error is thrown when there is a Gemfile.lock present specifying a ruby gem version. I guess this can be related to the issue @ilario is having. Maybe if you have previously installed ruby or the required gems in a subfolder of the git repo then ruby in the docker container considers what is present in the  ruby subfolder. 
It can be interesting to clone the repo in a new folder and start the docker container there.
thanks everybody for the feedback :)

**Comment by svalo** [comment]
@brunovianna [Here](https://github.com/svalo/lime-web/blob/69f466d64c003201f98b7330bc2779d38e3c0a85/README#L29) I mentioned that, do you think it should be pointed more cleary that one should move to the top-level directory of the cloned repo?

**Comment by brunovianna** [comment]
You're right. I missed it somehow. I guess it's good enough, I just wasn't very thorough reading the instructions.

File: README [hunk 1]
```diff
@@ -22,3 +22,20 @@ In Ubuntu/Debian you would need to install ruby header files
...
+An image from the jekyll project is available and documented here: https://github.com/jekyll/docker/.
+All you need to do -provided you have configured docker- is to open a terminal
+and move to the folder where you cloned this repo and issue:
+
+    $ export JEKYLL_VERSION=3.5
+    docker run --rm \
+        --publish 4000:4000 \
+        --volume="$PWD:/srv/jekyll" \
+        -it jekyll/jekyll:$JEKYLL_VERSION \
+        jekyll serve
```
Comment (nicopace) [inline README]: After a failed download it got broken.
This is the outcome:
```
$ docker run --rm         --publish 4000:4000         --volume="$PWD:/srv/jekyll"         -it jekyll/jekyll:$JEKYLL_VERSION         jekyll serve
Bundler can't satisfy your Gemfile's dependencies.
Install missing gems with `bundle install`.
```

Comment (nicopace) [reply README]: now it worked!
only some text changes, but ready to go!

**Comment by nicopace** [comment]
Thanks @svalo !!

**Comment by svalo** [comment]
Thank you all for the feedback :smiley:

**Pull request merged**